xml_inscriptions/transcribed/MA.Bos.MFA.G.67.743.xml

```diff
@@ -0,0 +1,176 @@
+                  <origin>
+                     <date evidence="#unspecified" notBefore="-0451" notAfter="-0449">about 450
+                        B.C.</date>
+                     <placeName ref="europe.greece.boiotia">Thebes</placeName>
+                  </origin>
+                  <provenance type="transferred" subtype="given">
+                     <p> gift of <persName role="donor">Dr. Elie Borowski</persName> to <orgName role="recipient">MFA</orgName>, <date when="1967">June 28, 1967</date></p>
+                  </provenance>
+                  <provenance type="found" subtype="first-recorded">
+                     <p> said to come from the Kabeirion, <placeName>Thebes</placeName></p>
+                  </provenance>
+                  <provenance type="observed" subtype="reported">
+                     <p> By date unknown: with <persName role="owner">Dr. Elie Borowski</persName>, Angensteinerstrasse 7, <placeName>Basel, Switzerland</placeName></p>
+                  </provenance>
+                  
+                  
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CVB"/>
+                  <biblScope unit="item">no. 58</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="https://collections.mfa.org/objects/152698/votive-bull?ctx=b335d544-0c0d-4a47-a166-2d1a8b03d3b5&amp;idx=0"
+                     >MFA web entry</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change who="Jean Wanlass" when="2023-07-12">added metadata, transcription</change>
+         <change who="Jean Wanlass" when="2023-07-25">minor corrections</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.67.743.jpg"/>
+      </surface>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.67.743.det.jpg"/>
+      </surface>
+   </facsimile>
+   
+   <text>
+      <body>
+         <div type="edition">
+            <ab>
+               <lb n="1"/>ΗΙRΟΣΤΟΡΑΔΑΟΙΣ 
```

Comment:
perhaps see if there's some more information available about how others have interpreted this?